great_expectations/rule_based_profiler/types/data_assistant_result/plot_result.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from typing import List
+
+import altair as alt
+
+
+@dataclass(frozen=True)
+class PlotResult:
+    """Wrapper object around DataAssistantResult plotted Altair charts.
+
+    Please note that contained within this object are the raw Altair charts generated
+    by `DataAssistantResult.plot()`. They may have been concatenated or formatted for
+    purposes of display in Jupyter Notebooks.
+
+    Attributes:
+        charts: The list of Altair charts rendered through `DataAssistantResult.plot()`
+    """
+
+    charts: List[alt.Chart]
```

Comment:
let's say we wanted to encapsulate our use of altair. Could we produce a property for "to_json" and "to_dict" and keep the chart itself private? </bad_idea>?

Reply:
hmm we definitely could but I think users are going to want to have the Altair Python objects since they can be directly updated during runtime. Some may just use the JSON output but I think we need to enable multiple workflows (even if that means exposing the library choice).

I think this is slightly different from our YAML situation in that this is recognized, well-supported library with a lot of recognition in the data space.